src/serverdlg.cpp

```diff
@@ -157,24 +157,25 @@ CServerDlg::CServerDlg ( CServer*         pNServP,
     // recorder status
     lblRecorderStatus->setAccessibleName ( tr ( "Recorder status label" ) );
     lblRecorderStatus->setWhatsThis ( "<b>" + tr ( "Recorder Status" ) + ":</b> "
-        + tr ( "Displays the current status of the recorder.  The following values are possible:"
-               "<dl>"
-               "<dt>Not initialised</dt>"
-               "<dd>No recording directory has been set or the value is not useable</dd>"
-               "<dt>Not enabled</dt>"
-               "<dd>Recording has been switched off" )
+        + tr ( "Displays the current status of the recorder.  The following values are possible:" )
+        + "<dl>"
+        + "<dt>" + tr ( SREC_NOT_INITIALISED ) + "</dt>"
```

Comment:
These are defined in `serverdlg.h` untranslated and translated in the two places each string is used.  I couldn't immediately spot a similar example to copy, so I'm not certain this is the best approach - open to suggestions.

Reply:
The Qt Linguist tool automatically finds the same text and uses the same translation. So please do not use the #defines but simply put the same text on the different locations in the code.

Reply:
BTW: What does <dl> and <dt> do? In the other places in Jamulus we always use <ul> and <li>. Would that be an option for you, too? Just for consistency.

Reply:
The use of the #define makes sure it actually _is_ the same text.  It gets expanded and translated (by Qt Linguist) in each place.  I'd rather keep it for consistency.  (Had there been an _actual_ status, then there would have been a method returning the string based on the status and each status could have been passed to that retrieve the string.)

`<dl>` is hypertext for "definition list".  This is a list of definitions of what the statuses mean.  So it's the appropriate list type.  It's not an "unordered list" (`<ul>`), which just gives a series of bullet points, or a "simple list" (`<sl>`).  `<dt>` is the term being defined and `<dd>` is the definition of the term.  The formatting this results in makes it easier to see and any accessibility tools will find it easier to handle the content.

Reply:
So do you recommend to replace all the `<ul>` by `<dd>` then?

Reply:
It depends on context.  If something is "just a list of items", then an unordered list makes sense.

For instance, saying features are:
* it does this
* it also does this

that's an unordered list.  But if you're explaining a list of terms, `<dl>` seems more appropriate than, say
* this term
means this
* another term
means something else

which is what you might do with `<ul>` to get the same effect.  The bullet doesn't feel right in the context.

The "Buffers" LED uses `<ul>` to supply a list of reasons - so it makes sense there - it's not defining anything.

The "Audio Channels" control uses `<ul>` -- but it's neither a definitions of terms from the dropdown nor even really a list of things.  So, whilst there are three entries in the dropdown and three bullets, the bullets aren't directly addressing each in turn, they are just a collection of comments generally addressing what they mean.  The use of `<ul>` at all here is almost uncalled for - just separate paragraphs would do.

Currently, on the server dialog, the server registration status doesn't list the strings that could appear at all (I hadn't thought of any way to keep them consistent easily, so didn't do it!), just makes a vague comment about what the field is for (which may help or may not...).

Reply:
Ok, makes sense.